pydatastructs/miscellaneous_data_structures/queue.py

```diff
@@ -64,7 +69,9 @@ def popleft(self, *args, **kwargs):
 
     @property
     def is_empty(self):
-        return None
+        raise NotImplementedError(
+            "This is an abstract method.")
+
```

Comment:
`return None` was used to show the abstractness. I think the change is fine to let the user know they are trying to access something that isn't defined.

Reply:
Also, it improves Codecov.